spec/app/ics-101-interchain-swap/README.md

```diff
@@ -117,68 +117,70 @@ The pool can be fully funded through the initial deposit or through subsequent d
 
 ```ts
 interface Coin {
-    amount: int64;
-    denom: string;
+  amount: int32;
+  denom: string;
 }
 ```
 
 ```ts
-enum PoolSide {
-  Native = 1;
-  Remote = 2;
+enum PoolAssetSide {
+  Source = 1;
+  Destination = 2;
 }
 ```
 
 ```ts
 // PoolStatus defines if the pool is ready for trading
 enum PoolStatus {
-  POOL_STATUS_INITIAL = 0;
-  POOL_STATUS_READY = 1;
+  INITIALIZED = 0;
+  ACTIVE = 1;
 }
 ```
 
 ```ts
 interface PoolAsset {
-    side: PoolSide;
-    balance: Coin;
-    // percentage
-    weight: int32;
-    decimal: int32;
+  side: PoolAssetSide;
+  balance: Coin;
+  // percentage
+  weight: int32;
+  decimal: int32;
```

Comment:
I am thinking if we could remove this field. since we don't use it al all

Reply:
okay

Reply:
weight can be used for calculation. maybe decimal is useful in frontend side to display assets